src/gallery.tsx

```diff
@@ -24,7 +24,7 @@ class Gallery extends React.Component<NoProps, NoState> {
         display: 'flex',
         flexDirection: 'row'
       } as CSSProperties;
-      return <div style={style} key={it.globalCardId}>
+      return <div style={style} key={it.globalCardId.id}>
```

Comment:
Should this be asString()? `id` is not unique... ;)

Reply:
it should and now is :)